packages/core/src/Stores/client-store.js

```diff
@@ -2502,5 +2509,17 @@ export default class ClientStore extends BaseStore {
             await this.switchAccount(this.virtual_account_loginid);
         }
     }
+
+    setIsPreAppStore(is_pre_appstore) {
+        const trading_hub = is_pre_appstore ? 1 : 0;
+        WS.setSettings({
+            set_settings: 1,
+            trading_hub,
+        }).then(response => {
+            if (!response.error) {
+                this.account_settings = { ...this.account_settings, trading_hub };
+            }
+        });
+    }
```

Comment:
Could we have a catch block to handle error or console error information.?

Reply:
since it's a ui change and user couldn't see the expected changes if we get error in response, she/he can simply try one more time to get the expected result

Reply:
Yep, The reason behind the suggestion was  it could improve the code maintainability, It is fine to omit now. if it is may not effect user experience.